### DISCUSSION TITLE
WalletConnect - peerMeta data

### DISCUSSION THREAD

**Discussion by imqdee** [wevm/wagmi#275] [Q&A]
Hi makers 👋 

I integrated WalletConnect using `wagmi` days ago. Recently I caught that the peerMeta field isn't populated, the wallet I connect to when using WalletConnect (Ledger Live) isn't able to get the name or the description of my dapp.
I read from the WalletConnect documentation ([here](https://docs.walletconnect.com/tech-spec#walletconnect-methods)) that these information should be scrapped from the loaded webpage's head meta-tags

> In browser applications, this is scraped from the loaded webpage's head meta-tags. In native applications, this is provided by the application developer.

Am I missing something or is it something wagmi should implement this logic? If so, could you give me some clues on the best way to do it? Would be happy to contribute!

Thanks!

**Comment by tmm** [comment]
Hi! Seems like something that needs to be set up on Ledger's end. Here's an [example of how Rainbow sets up WalletConnect meta](https://github.com/rainbow-me/rainbow/blob/f159380de466658ef22ad475611b58d9683ab505/src/redux/walletconnect.js#L78-L95=).
